app/controllers/concerns/state_file/state_file_controller_concern.rb

```diff
@@ -1,52 +1,27 @@
+# TODO i left these things in here because this concern exists but the thing is the concern is only included once (in the state file questions controller) so should we just move all this over to the controller itself?
 module StateFile
   module StateFileControllerConcern
     extend ActiveSupport::Concern
 
     included do
-      helper_method(
-        :current_tax_year, :filer_count, :state_name, :state_abbr, :ny?, :az?, :state_param
-      )
+      helper_method :current_tax_year, :state_name, :state_code
     end
 
-    private
-
-    def state_name
-      States.name_for_key(state_abbr)
-    end
-
-    def state_abbr
-      state_param&.upcase
-    end
-
-    def state_param
-      params[:us_state]
-    end
-
-    def ny?
-      state_param == "ny"
-    end
-
-    def az?
-      state_param == "az"
-    end
-
-    def service_type
-      case state_param
-      when "az" then :statefile_az
-      when "ny" then :statefile_ny
+    # TODO idea: state_code could prioritize getting the code from the current intake and if there isn't one, default to the params
```

Comment:
i _think_ this idea is out of scope for this story but i wanted to leave this comment because the idea of a controller helper method called something as generic as `state_code` that only looks at the params feels incomplete to me